tests/test_lammpstrj.py

```diff
@@ -67,7 +67,7 @@ def test_read_write_0():
         xyz2, new_lengths, new_angles = f.read()
 
     eq(lengths, new_lengths)
-    eq(angles, new_angles)
+    eq(angles, new_angles, decimal=3)
```

Comment:
Hey Jeremy, I was very interested in this test in particular since the way it has been fixed has been to basically increase the allowed difference between arrays. I did an investigation of the test and also looked at the effect of the data type in `LAMMPSTrajectory`, see here: https://github.com/janash/lammps_mdtraj_comparison

I think we may want to want to increase the data type in LAMMPS Trajectory to be `np.float64`. I don't really understand why it is `np.float32`.

Reply:
+1 for using `float64` anywhere it keeps LAMMPS tests passing

Reply:
I agree with using float64 as well. That's a very comprehensive repo @janash! 

I dug through the lammps code just a little (I never used it before), and it's funny that [all angles writes/calculations are essentially double](https://github.com/lammps/lammps/blob/59bbc5bcc1104bdb4fb45107cd65b5d4d76dbc00/src/angle_write.cpp) everywhere I looked anyways.

Reply:
I got a little preoccupied with the test, I suppose 😇 

As far as the data type - `np.float64`, vs `np.float32` - is it that `np.float64` is the default data type for an array if not specified? (Like `np.ones(100)` will be `np.float64`?). Is there a default float type? If there is, was it different in the past/do we expect it to be different in the future?

Reply:
> I got a little preoccupied with the test, I suppose 😇
> 
> As far as the data type - `np.float64`, vs `np.float32` - is it that `np.float64` is the default data type for an array if not specified? (Like `np.ones(100)` will be `np.float64`?). Is there a default float type? If there is, was it different in the past/do we expect it to be different in the future?

Default's np.float64, it seems
https://numpy.org/doc/stable/reference/arrays.dtypes.html#specifying-and-constructing-data-types

Reply:
@jeremyleung521 - also great to know that a need for `np.float32` isn't coming from anywhere in the LAMMPS writer

Reply:
The new commit below (4ca025a) should effectively swap over to `np.float64` for `LAMMPSTrajectory`'s angles, lengths and xyz.